src/dotdom.js

```diff
@@ -37,7 +37,7 @@ module.exports = window;
    * @param {Array} [children] - The child VNode elements
    * @returns {VNode} Returns a virtual DOM instance
    */
-  let createElement = (element, props={}, ...children) => (
+  let c = (element, props={}, ...children) => (
```

Comment:
The insane scope requirement is to handle a side effect of javascript where variables declared at the top level ( not in a function block ) are implicit global variable definitions.

This IIFE wrapper could be added at build/minification time, or just wrapped around the whole file from the start. The closure also ensures that now var references are leaked.

@SilentCicero your byte fetishism is deep!
software engineering is a balance of many dimensions